### DISCUSSION TITLE
Working groups should have 3 co-chairs not 2

### DISCUSSION THREAD

**Discussion by jawache** [Green-Software-Foundation/oc#9] [General]
The truth is our charter only allows for a chair + vice-chair, when we moved to co-chairs we didn't update our charter so regardless of the discussion here, we need to update our charter.

I propose we change our charter to support 3 co-chairs for every WG.

- Most other groups either have chair + vice-chair OR 3 co-chairs, to ensure that there is a clear path forward and no split decisions.
- Our choice of 2 was more just an evolution of chair+ vice-char and didn't give much thought to split decisions.
- The foundation is growing fast, we need to grow more leaders to support our growth.
- There were more than a handful of WG meetings had to be cancelled last year due to both chairs not being available at the last minute, 3 chairs will mitigate that risk and make meetings more consistent.
- 3 chairs will allow more diversity in geography and language, perhaps we should insist that all 3 chairs are not in the same continent (e.g. EMEA, APAC, Americas) to open up the possibility of meetings being held in different times to accommodate more of the global population.

**Comment by tmcclell** [comment]
Agree

**Comment by Henry-WattTime** [comment]
I think of chairs as more facilitators than decision makers? Am I interpreting that correctly?

**Comment by Sealjay** [comment]
Agreed with having three co-chairs; not as fussed about keeping the continent as optional.

Usually we only need timezones from at least two thirds of the world. E.g. someone in Europe can cover North America/LatAm + Europe **or** Europe + Asia Pacific/other timezones, etc etc.

**Comment by tmcclell** [comment]
Do we have interest to fill 3 chairs per WG?
